bodhi/server/validators.py

```diff
@@ -414,8 +414,8 @@ def validate_acls(request):
                 request.errors.add('body', 'builds', error_msg)
                 return
         elif acl_system == 'dummy':
-            people = (['ralph', 'bowlofeggs', 'guest'], ['guest'])
-            groups = (['ralph', 'bowlofeggs', 'guest'], ['guest'])
+            people = (['ralph', 'bowlofeggs', 'guest', 'mcurlej'], ['guest'])
+            groups = (['ralph', 'bowlofeggs', 'guest', 'mcurlej'], ['guest'])
```

Comment:
It would be good to put this in its own independent commit, since it's not related to the duplicate test cases.

Also, we should probably eventually make this a setting in ```development.ini```, but I'm fine with doing it this way for now unless you want to take that on ☺